### ISSUE TITLE
Time to first response: Avoiding container cold start

### ISSUE THREAD

**Issue by felixcremer** [EarthyScience/FAIRSenDD#14]
Should we have a running julia process that can get some request and immidiately runs the rqatrend computations or do we want to always launch a new julia process which would have some overhead

**Comment by danlooo** [comment]
Indeed, always keeping a warm container will result into much faster responses for a small initial request, circumventing spin-up time. However, there is no such thing as `POST job` in OGC API - Processes. We always have a new function call in pygeoapi for each job execution. Thus, we need to keep the workers warm in `Distributed.jl` or `Dagger.jl`.
